### PR TITLE
TOC Headline Caption added

### PR DESCRIPTION
**Pull request by bernardosimonetto**
Added Table of Content attributes captionText and useDottedLines.

### CODE REVIEW

File: src/WkHtmlToPdf-DotNet/Settings/TableOfContentsSettings.cs [hunk 1]
```diff
@@ -2,12 +2,29 @@ namespace WkHtmlToPdfDotNet
...
+        /// Name/Headline of the TOC
+        /// </summary>
+        [WkHtml("toc.captionText")]
+        public string CaptionText { get; set; }
 
+        /// <summary>
+        /// Should we print dots between the name and the page number?
+        /// </summary>
+        [WkHtml("toc.useDottedLines")]
+        public bool	UseDottedLines { get; set; }
```
Comment (HakanL) [inline src/WkHtmlToPdf-DotNet/Settings/TableOfContentsSettings.cs:21]: Fix the formatting here (extra spaces/tabs)

File: src/WkHtmlToPdf-DotNet/Settings/TableOfContentsSettings.cs [hunk 2]
```diff
@@ -2,12 +2,29 @@ namespace WkHtmlToPdfDotNet
...
+        
         public TableOfContentsSettings()
         {
             IsTableOfContent = true;
+            CaptionText = "Table of Contents";
```
Comment (HakanL) [inline src/WkHtmlToPdf-DotNet/Settings/TableOfContentsSettings.cs:26]: I don't think we should set these defaults, please remove, empty string and false should be fine.

File: src/WkHtmlToPdf-DotNet/Settings/TableOfContentsSettings.cs [hunk 3]
```diff
@@ -2,12 +2,29 @@ namespace WkHtmlToPdfDotNet
...
+            UseDottedLines = true;
         }
     }
-}
```
Comment (HakanL) [inline src/WkHtmlToPdf-DotNet/Settings/TableOfContentsSettings.cs:13]: All lines should end with NewLine, please revert the change to this line

**Review by HakanL — CHANGES_REQUESTED** [review]
Please see inline for changes

File: src/WkHtmlToPdf-DotNet/Settings/TableOfContentsSettings.cs (hunk 1 above)
Comment (bernardosimonetto) [reply src/WkHtmlToPdf-DotNet/Settings/TableOfContentsSettings.cs]: changed tab to space in line 21 and removed tabs in empty lines

File: src/WkHtmlToPdf-DotNet/Settings/TableOfContentsSettings.cs (hunk 2 above)
Comment (bernardosimonetto) [reply src/WkHtmlToPdf-DotNet/Settings/TableOfContentsSettings.cs]: removed constructor

File: src/WkHtmlToPdf-DotNet/Settings/TableOfContentsSettings.cs (hunk 3 above)
Comment (bernardosimonetto) [reply src/WkHtmlToPdf-DotNet/Settings/TableOfContentsSettings.cs:13]: added new line at file end

**Comment by HakanL** [comment]
Since the requested changes weren't made I will close this PR. But I have added the TOC changes to master.
